pandora-client-web/src/components/wiki/pages/spaces.tsx

```diff
@@ -170,6 +170,14 @@ export function WikiSpaces(): ReactElement {
 				<li>If you want to permanently switch to another chat mode via a chat command, send the command without a text message behind the command.</li>
 				<li>Certain chat modes can be combined with the whisper mode.</li>
 				<li>All chat modes have a variant to send text without any formatting, which are the "raw" variants of the according chat commands.</li>
+				<li>
+					There are also shortcuts to write one-off messages while in the default, normal chat mode.<br />
+					These shortcuts are<br />
+					"((" - (( ooc message ))<br />
+					"*"  - *me-emote that gets prefixed with your character's name*<br />
+					"**" - **generic emote that doesn't get prefixed**<br />
+					Note: A single message can have multiple parts, so the above shortcut types can be mixed in a single message if each different one is in a new line.
```

Comment:
not sure how detailed we want to be but to be precise

formatting and modifiers do not need to be closed, they are closed automatically

in addition to the new line, the previous modifier must also be closed

Reply:
Added